eng/pipelines/templates/jobs/installation.check.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: "not-specified"
+
+jobs: 
+  - job: InstallationCheck
+    displayName: "Installation Check"
+    variables:
+      - template: ../variables/globals.yml
+    steps:
+      - download: current
+        artifact: ${{parameters.ArtifactName}}-signed
+      - pwsh: |
+          Write-Host "Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "$(Pipeline.Workspace)/install-check-dir/${{parameters.Artifact.name}} -Recurse"
+          Copy-Item "${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "install-check-dir/${{parameters.Artifact.name}}" -Recurse
+          Write-Host "$pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom"
+          $pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom
```

Comment:
Do we need to have the file call pom.xml? Can we just use the pom file as it is named?

Reply:
I think we could set the mavenPOMFile property in maven task, I'll need to test that out. Or else the default behavior is to run on pom.xml